themes/nav-community-v1/layouts/partials/head.html

```diff
@@ -16,19 +17,17 @@
     <title>{{ .Title }}</title>
 
     <!-- google font -->
-    <link href="https://fonts.googleapis.com/css?family=Patua+One" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Patua+One|Roboto+Slab" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Patua+One|Roboto+Slab|Raleway|Roboto" rel="stylesheet">
 
     <!-- bootstrap -->
     <link rel="stylesheet" href="{{ "css/bootstrap.min.css" | relURL }}">
 
     <!-- google fonts -->
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway|Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=">
```

Comment:
Why is this font been removed?

Reply:
It hasn't, I just added it to the import link on line 20.